car-rental/src/main/java/com/fiap/techchallenge/carrental/controller/ClienteController.java

```diff
@@ -56,6 +56,6 @@ public ResponseEntity<String> deletarCliente(@PathVariable long numeroCnh) {
             LOGGER.error("Não foi possível eliminar o cliente {}!", numeroCnh);
             return new ResponseEntity<>("Não foi possível eliminar o cliente!", HttpStatus.INTERNAL_SERVER_ERROR);
         }
-        return new ResponseEntity<>("Cliente eliminado com sucesso!", HttpStatus.INTERNAL_SERVER_ERROR);
+        return new ResponseEntity<>("Cliente eliminado com sucesso!", HttpStatus.ACCEPTED);
```

Comment:
```suggestion
        return new ResponseEntity<>("Cliente eliminado com sucesso!", HttpStatus.OK);
```

Pesquisei o status 202 (ACCEPTED) e achei o 200 mais aceitavel nesse caso, pois o processo é executado e finalizado. Segue fonte:

**202 Accepted**

The HyperText Transfer Protocol (HTTP) 202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202